### PR TITLE
EPMRPP-38114 React. Dashboards mobile navigation

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  dashboards: dashboardItemsSelector(state),
+  activeItemId: activeDashboardIdSelector(state),
+}))
+@injectIntl
+export class DashboardPageHeader extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    projectId: PropTypes.string.isRequired,
+    activeItemId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    dashboards: PropTypes.array,
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx]: Let's make it easier to understand what we expect here:
`PropTypes.arrayOf(Prop.Types.shape(...))`

File: app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { NavLink } from 'redux-first-router-link';
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx]: Link

File: app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  render() {
+    const { activeItemId, eventsInfo } = this.props;
+
+    return (
+      <div className={cx('dashboard-page-header')}>
+        <div className={cx('dashboards-nav-list-mobile')}>
+          <InputDropdown
+            options={this.generateOptions()}
+            value={activeItemId || DASHBOARD_PAGE_ITEM_VALUE}
+            onChange={this.onChangeTab}
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx]: `this.onChangeTab` doesn't exist

File: app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+.dashboards-nav-list-mobile {
+  display: none;
+  @media (max-width: $SCREEN_XS_MAX) {
+    display: block;
+    padding: 13px 10px;
+    background: $COLOR--white-two;
+    span {
+      a {
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.scss]: Is it really necessary to write nested tag selectors?

File: app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx (hunk 2 above)
Comment (AmsterGet) [reply app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx]: Is it a actually `NavLink` here, isn't it?

File: app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx (hunk 3 above)
Comment (AmsterGet) [reply app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx]: Fixed

File: app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.scss (hunk 4 above)
Comment (AmsterGet) [reply app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.scss]: Added `activeClassName` to `NavLink`

File: app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.jsx]: Fixed

**Pull request merged**